org.omegat.OmegaT.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<component type="desktop">
+	<id>org.omegat.OmegaT</id>
+	<name>OmegaT</name>
+	<summary>A simple, open-source translation memory manager and CAT tool</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	<description>
+		<p>OmegaT is a free translation memory application that works on Windows, macOS, Linux... It is a tool intended for professional translators. It does not translate for you! (Software that does this is called "machine translation", and you will have to look elsewhere for it.)</p>
+	</description>
+	<url type="homepage">https://omegat.org/</url>
+	<project_license>GPL-3.0</project_license>
+	
+	<content_rating type="oars-1.0">
+		<content_attribute id="social-info">mild</content_attribute>
+	</content_rating>
+
+	<developer_name>The OmegaT Project</developer_name>
+	<releases>
+		<release date="2021-04-07" version="5.4.3" />
+	</releases>
+	
+	<screenshots>
+		<screenshot type="default">
+			<image>https://omegat.org/img/screenshots/main_window_en.png</image>
+		</screenshot>
+		<screenshot>
+			<image>file://omegat-screenshot.png</image>
```

Comment:
`file:` URL are not allowed in appstream (on flathub)

That's why the build failed.